src/resources/styles-data.ts

```diff
@@ -0,0 +1,155 @@
+export const darkStyles = {
```

Comment:
We should split dark and derived styles, so only the derived styles will be in the entrypoint